.github/workflows/release.yml

```diff
@@ -62,15 +62,15 @@ jobs:
         with:
           token: ${{ secrets.GH_TOKEN }}
           prepend-product-to-tag: true
-      - name: Update Pages
-        uses: ./.github/.release/actions/actions/services/github-pages
-        with:
-          token: ${{ secrets.GH_TOKEN }}
-          source-folder: build/dokka/gfmMultiModule
-          jira-api-key: ${{ secrets.JIRA_API_KEY }}
-          last-service: true
-      - name: Upload test reports
-        uses: ./.github/.release/actions/actions/test-reports/upload
-        with:
-          token: ${{ secrets.GH_TOKEN }}
-          acceptance-tests-workflow: Tests
+#      - name: Update Pages
```

Comment:
I am wondering why we need this in kotlin SDK and not in kmp-chat?

Reply:
it's for publishing KDocs to Github pages, it works in pubnub-kotlin but something is not configured in kmp-chat project on GH so it fails. Until we can get someone to configure kmp-chat (enable github pages?) then let's keep this disabled so the release can complete successfully

Reply:
Thanks for clarification.